uptrain/operators/language/llm.py

```diff
@@ -81,6 +82,16 @@ async def async_process_payload(
                     and count < max_retries - 1
                 ):
                     # refer - https://github.com/BerriAI/reliableGPT/
+                    # if required to set token limit - https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/chatgpt?pivots=programming-language-chat-completions#managing-conversations
+                    
+                    fallback = {
+                        "gpt-3.5-turbo": "gpt-3.5-turbo-16k",
+                        "gpt-3.5-turbo-0613": "gpt-3.5-turbo-16k-0613",
+                        "gpt-4": "gpt-4-32k",
+                        "gpt-4-0613": "gpt-4-32k-0613",
+                    }
+
+                    payload.data["model"] = fallback[payload.data["model"]]
                     # TODO: check the model being used and not always switch to 3.5 16k variant
                     payload.data["model"] = "gpt-3.5-turbo-16k"
```

Comment:
@Dominastorm - I guess this line should be deleted?

Reply:
Yeah sorry, missed that. Fixed it now